wgsl/index.bs

```diff
@@ -223,6 +223,9 @@ A shader comprises:
 * The set of variables and constants [=statically accessed=] by all those functions.
 * The set of types used to define or analyze all those functions, variables, and constants.
 
+Note: A WGSL program does not require an [=entry point=]; however, such a
+program cannot be executed by the API.
```

Comment:
I wonder if it would help readers more if this section elaborated that's due to no possible GPUProgrammableStage than just saying API cannot execute it, space trade-off might be worth it.

Reply:
New commit is great